### PR TITLE
fixed navitia service docblocks

### PR DESCRIPTION
**Pull request by datanel**

### CODE REVIEW

File: Services/Navitia.php [hunk 1]
```diff
@@ -24,37 +24,39 @@ public function setToken($token)
...
      *
-     * @param  type $externalCoverageId
-     * @param  type $networkId
-     * @return type
+     * @param string $externalCoverageId
+     * @param string $networkId
+     *
+     * @return array
      */
     public function getLines($externalCoverageId, $networkId, $depth = 0, $count = false)
```
Comment (bbonnesoeur) [inline Services/Navitia.php]: Now you've clean every docblock, could you add misssing depth and count param here ? ;)

File: Services/Navitia.php [hunk 2]
```diff
@@ -150,41 +166,45 @@ public function getStopPoints($coverageId, $networkId, $lineId, $routeId)
...
+
         return $this->navitia_component->call($query);
     }
 
     /**
-     * Get one Stop Point
+     * Get one Stop Point.
+     *
+     * @param string $coverageId
+     * @param string $networkId
```
Comment (bbonnesoeur) [inline Services/Navitia.php]: wrong var name $networkId instead of $stopPointId in method signature

File: Services/Navitia.php [hunk 3]
```diff
@@ -193,29 +213,29 @@ public function getNetworkModes($coverageId, $networkId, $commercial = true)
...
 
         return $this->navitia_component->call($query);
     }
 
     /**
-     * Returns Basic Route data with corresponding line data
+     * Returns Basic Route data with corresponding line data.
      *
-     * @param  String $coverageId
-     * @param  String $routeId
```
Comment (bbonnesoeur) [inline Services/Navitia.php]: missing $depth dockblock

File: Services/Navitia.php (hunk 1 above)
Comment (datanel) [inline Services/Navitia.php]: fixed and I have changed $depth = false to $depth = null

File: Services/Navitia.php (hunk 3 above)
Comment (datanel) [inline Services/Navitia.php]: fixed

File: Services/Navitia.php (hunk 2 above)
Comment (datanel) [inline Services/Navitia.php]: fixed

**Pull request merged**